### ISSUE TITLE
Poor contrast for default reporter Success message

### ISSUE THREAD

**Issue by sirbrillig** [atoum/atoum#762]
The default `Success` message has poor contrast (gray over green).

<img width="616" alt="screen shot 2017-10-09 at 7 05 16 pm" src="https://user-images.githubusercontent.com/2036909/31362113-cf6cdf2e-ad24-11e7-9913-c082f557eb84.png">

**Comment by Hywan** [comment]
Actually, the foreground color is black (`30`). What terminal or color palette are you using?

**Comment by jubianchi** [comment]
<img width="568" alt="capture d ecran 2017-10-10 a 06 25 16" src="https://user-images.githubusercontent.com/327237/31381907-e8789ad0-ad83-11e7-80bf-546e2476a55e.png">

seems OK: the foreground is black.

**Comment by sirbrillig** [comment]
🤔 Ah, I think I found it. It's black, but it's "bright" black. In my terminals, bright black is a lighter color, and I think that's the default (MacOS iTerm and MacOS Hyper). Could we just make it regular black? (I clearly don't have a lot of experience with terminal colors, so maybe I'm misunderstanding something important.)

**Comment by Hywan** [comment]
Right now, it's bold + black. Maybe we should go back to black only?

**Comment by jubianchi** [comment]
@sirbrillig you are right, it's bright black (`1;30`). Will change it to plain black (`0;30`)

**Comment by guiled** [comment]
To help work on it, this is related to https://github.com/atoum/atoum/pull/704
